how-it-works/2_chainkeytech/01_chain-key-technology.subpage.md

```diff
@@ -1,16 +1,36 @@
 ---
-title: Chain Key Cryptography
+title: Chain-Key Cryptography
```

Comment:
It would be great with a diagram in this text or the overview - but perhaps this is not that easy to accomplish in markdown?